src/LinSys/LinSysMerger.h

```diff
@@ -1,7 +1,7 @@
 // *****************************************************************************
 /*!
   \file      src/LinSys/LinSysMerger.h
-  \author    J. Bakosi
+  \author    Los Alamos National Laboratory
```

Comment:
These are good points, including the blog post. However, adding an author field as "Los Alamos National Laboratory" has completely zero information content.